src/test/java/com/github/onsdigital/dp/authorisation/permissions/CheckerTest.java

```diff
@@ -63,7 +63,7 @@ public void setUp() throws Exception {
     @Test
     public void testHasPermission() throws Exception {
         Boolean hasPermission = checker.hasPermission(
-                new UserDataPayload("userId", new String[]{"admin"}), "users.add", null
+                new UserDataPayload("userId", "userEmail", Arrays.asList(new String[]{"admin"})), "users.add", null
```

Comment:
We don't really need to create an array when using the `Arrays.asList` and could use var args:
```suggestion
                new UserDataPayload("userId", "userEmail", Arrays.asList("admin")), "users.add", null
```

Reply:
Fixed redundant array creation for calling varargs method.
Please review it again, thanks for the suggestion.